packages/sdk-js/src/GrowthBook.ts

```diff
@@ -306,7 +306,12 @@ export class GrowthBook<
     const valueHash = JSON.stringify(result.value);
 
     // If the changes are already active, no need to re-apply them
-    if (result.inExperiment && existing && existing.valueHash === valueHash) {
+    if (
+      result.inExperiment &&
+      existing &&
+      existing.valueHash === valueHash &&
+      !forced
```

Comment:
This is probably the most risky change here - we previously did not leverage `forced` here (even though we had it available in this function). This might change prior behavior